packages/shared/forks/ReactFeatureFlags.www.js

```diff
@@ -122,5 +122,7 @@ export const disableLegacyMode = true;
 export const enableOwnerStacks = false;
 export const enableShallowPropDiffing = false;
 
+export const enableUseResourceEffectHook = true;
```

Comment:
You probably want to make this a dynamic flag with a killswitch since it could impact existing effect behavior.